.github/workflows/shippable_builds.yml

```diff
@@ -678,11 +689,18 @@ jobs:
             esac
           fi
 
+          if [[ "${RELEASE_TYPE}" = "daily" ]]; then
+            FTPMO_SUBDIR="nightly"
+          else
+            FTPMO_SUBDIR="releases"
+          fi
+
```

Comment:
You could simply do `${{ needs.dump_config.outputs.releaseType == 'daily' && "nightly" || "release" }}` in the GCS step. This isn't really package info, so it isn't really the right step.